src/data/actions/pingStudio.js

```diff
@@ -0,0 +1,19 @@
+export const PING_RESPONSE = 'PING_RESPONSE';
+
+function pingResponse(response) {
+  return {
+    type: PING_RESPONSE,
+    status: response.status,
+  };
+}
+
+export function pingStudio() {
+  return dispatch =>
+    fetch('/api/assets/course-v1:edX+DemoX+Demo_Course/?page=0&page_size=50&sort=sort&asset_type=', {
+      credentials: 'same-origin',
+      headers: {
+        Accept: 'application/json',
+      },
+    })
+      .then(response => dispatch(pingResponse(response)));
```

Comment:
Like in our `mapDispatchToProps` defined on `BackendStatusBanner`, we use `dispatch` to invoke an action creator. That's pretty much all `dispatch` does, and it's important that we only invoke action creators through `dispatch` so their resulting data is handled through Redux, without unexpected side effects. This is the power of one-way data flow -- single source of truth, no funky rando jQuery functions mutating data that's inaccessible to other UI. 

Within the `fetch` call's asynchronous promise chain, we invoke `pingResponse` passing in the HTTP response, so it can then be digested and stored by Redux.